Library/include/CSP/Services/WebService.h

```diff
@@ -105,14 +138,14 @@ class CSP_API ResultBase
 
 	/// @brief Get a code representing the failure reason, if relevant.
 	/// @return int
-	int GetFailureReason() const;
+	ERequestFailureReason GetFailureReason() const;
 
 protected:
 	ResultBase(csp::services::EResultCode ResCode, uint16_t HttpResCode);
 
 	void SetResult(csp::services::EResultCode ResCode, uint16_t HttpResCode);
 
-	virtual int ParseErrorCode(const csp::common::String& Value);
+	ERequestFailureReason ParseErrorCode(const csp::common::String& Value);
```

Comment:
Can we also make this private as it doesn't need to be accessed by any derived types?

Reply:
Just waiting on this to be done then I can approve this PR

Reply:
oops didn't see this one sneak in after the approval